fe/fe-core/src/main/java/com/starrocks/sql/analyzer/BackupStmtAnalyzer.java

```diff
@@ -168,7 +167,7 @@ public static void analyzeTableRef(TableRef tableRef, String dbName, Database db
         }
 
         if (partitionNames != null) {
-            if (tbl.getType() != Table.TableType.OLAP) {
+            if (!tbl.isOlapOrLakeTable()) {
                 ErrorReport.reportSemanticException(ErrorCode.ERR_WRONG_TABLE_NAME, tableName.getTbl());
```

Comment:
ERR_NO_LAKE_TABLE

Reply:
Maybe we should use the same error as before.

Reply:
I think we should keep the same error code as before.